src/lua/init.lua

```diff
@@ -349,13 +349,31 @@ local function run_preload()
     end
 end
```

Comment:
Nit: commit msg is a bit confusing, as one has to read title as a part of the continuing sentence (but the second line starts with a capital for whatever reason).  Feel free to ignore, but it would be much easier to read if reworded.

Reply:
Removed 'so'.